pkg/addons/manifest.go

```diff
@@ -185,7 +187,7 @@ func combineManifests(manifests []*bytes.Buffer) *bytes.Buffer {
 	return bytes.NewBufferString(strings.Join(parts, "\n---\n") + "\n")
 }
 
-func txtFuncMap(overwriteRegistry string) template.FuncMap {
+func txtFuncMap(overwriteRegistry string, caBundle bool) template.FuncMap {
```

Comment:
```suggestion
func txtFuncMap(overwriteRegistry string) template.FuncMap {
```

Let's avoid using such bools. Instead, let's use {{ if .Config.caBundle }} directly in template.

Reply:
@kron4eg Reverted that change, PTAL.